lib/LMSManagers/LMSHelpdeskManager.php

```diff
@@ -741,4 +741,11 @@ public function GetQueueCategories($queueid) {
 			JOIN rtcategories c ON c.id = qc.categoryid
 			WHERE queueid = ?', 'id', array($queueid));
 	}
+
+	public function GetTicketCategories($ticketid) {
```

Comment:
Ale LMSHelpdeskManager::GetTicketContents(...) nie pobiera nazwy kategorii, która jest potrzebna do powiadomienia. Chciałem w LMSHelpdeskManager::GetTicketContents(...) linijkę którą wskazałeś rozbudować do postaci dodanej metody ale zrezygnowałem bo stwierdziłem, że jest to metoda szeroko stosowana w całym systemie, więc jeśli zrobię dodatkowy join w tej metodzie tylko dla potrzeby powiadomień to niepotrzebnie będę zwiększał czas wykonania zapytania. Dlatego użyłem nowej metody